src/backend/controllers/PostController.ts

```diff
@@ -78,10 +78,17 @@ export const createPostHandler = function (schema, request) {
         }
       );
     }
-    const { postData } = JSON.parse(request.requestBody);
+
+    const content = request.requestBody.get("content");
+    const img = request.requestBody.get("img");
+
     const post = {
       _id: uuid(),
-      ...postData,
+      content,
+      img: img !== null ? window.URL.createObjectURL(img) : undefined,
```

Comment:
you can also try this statement as 
`img: img ? window.URL.createObjectURL(img) : undefined,`